finclab/my-wealth-frontend
# Title: verify user bvn number

#### What does this PR do?
- This PR implement the verify BVN feature for an authenticated user.

#### Description of Task to be completed?
- Implement verify user bvn feature.
- Implement user input validation.
- Implement tests for components, reducers and actions for the stated feature.

#### How should this be manually tested?
- Clone the repo.
- Cd into the project directory and run `npm install` to install all dependencies
- Run the react application by running `npm start`.
- Visit `http://localhost:3000/` on your browser
- Login as an authenticated user
- On the user dashboard click on the verify me drop down then click on BVN verification.
- Run `npm run test` to run test on your terminal or command line.

#### Any background context you want to provide?
<img width="1190" alt="Screen Shot 2019-05-01 at 1 39 42 PM" src="https://user-images.githubusercontent.com/32917511/57017742-5d30b980-6c18-11e9-9483-6a062ff51bca.png">


#### What are the relevant pivotal tracker stories?(if applicable)
[#165016024](https://www.pivotaltracker.com/story/show/165016024)
